tests/brocade/test_brocade_switch_protocol.py

```diff
@@ -1527,10 +1519,11 @@ def test_ip_redirect(self, t):
 
         remove_vlan(t, "1201")
 
+
 def enable(t):
     t.write("enable")
     t.read("Password:")
-    t.write_invisible(brocade_privileged_password)
+    t.write_invisible(t.conf["extra"].get("password", "root"))
```

Comment:
the other tests do not do this like this, why does this one?

Reply:
all other enable functions do it like this

Reply:
I meant why does the get on the password go to "root"

Reply:
because some switches don't specify a different password for the enable, so we just use the default root